nixos/modules/programs/firefox.nix

```diff
@@ -136,10 +244,19 @@ in
       };
 
     # Preferences are converted into a policy
-    programs.firefox.policies = mkIf (cfg.preferences != { }) {
+    programs.firefox.policies = {
```

Comment:
Why?

Reply:
The worst result is adding an empty item to the file. I thought it doesn't deserve to make the code more complicated.